Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest-results/ContestResultsPage.tsx

```diff
@@ -106,4 +109,4 @@ const ContestResultsPage = () => {
     );
 };
 
-export default makePrivate(setLayout(ContestResultsPage));
+export default makePrivate(asPage(setLayout(ContestResultsPage)));
```

Comment:
This should be done to all pages, abstract from them. i.e. in `PageContent.tsx`. 

Something like the following:
```
const PageContent = () => {
    const renderRoute = useCallback(
        ({path, Element}) => {
            const ElementAsPage = asPage(Element);
            return (
                <Route key={path} path={path} element={<ElementAsPage />} />
            );
    },
    [],
);

    return (
        <main className={styles.main}>
            <Routes>
                {routes.map(renderRoute}}
            </Routes>
        </main>
    );
);
```